airbyte-integrations/connectors/destination-weaviate/destination_weaviate/client.py

```diff
@@ -101,6 +136,7 @@ def flush(self, retries: int = 3):
                 logging.info(f"Object {obj_id} had errors: {errors}. Going to retry.")
 
         for buffered_object in self.objects_with_error.values():
+            print("Retrying buffered object", buffered_object.id, buffered_object.properties)
```

Comment:
could you use logging.info or logging.warn? I do think it's important we log these